client/SFTPClientGUI/SFTPClientGUI.py

```diff
@@ -70,102 +61,132 @@ def __init__(self, builder, *args, **kwargs):
+		col_size = Gtk.TreeViewColumn('Size')
+		col_size.pack_start(col_text, True)
+		col_size.add_attribute(col_text, 'text', 4)
 		col_size.set_sort_column_id(5)
-		col_date = Gtk.TreeViewColumn("Date Modified")
-		col_date.pack_start(colText, True)
-		col_date.add_attribute(colText, "text", 6)
+		col_date = Gtk.TreeViewColumn('Date Modified')
+		col_date.pack_start(col_text, True)
+		col_date.add_attribute(col_text, 'text', 6)
 		col_date.set_sort_column_id(6)
 		self.treeview_local.append_column(col)
 		self.treeview_local.append_column(col_perm)
 		self.treeview_local.append_column(col_size)
 		self.treeview_local.append_column(col_date)
 
-		self.treeview_local.connect('row-expanded', self.expand_row)
-		self.treeview_local.connect('row-collapsed', self.collapse_row)
+		self.treeview_local.connect('row-expanded', self.signal_expand_row)
+		self.treeview_local.connect('row-collapsed', self.signal_collapse_row)
 		self._tvmodel = Gtk.TreeStore(str, Pixbuf, str, str, str, int, str)
 
 		self.local_hidden = True
+		self._get_popup_menu()
+
+	def _get_popup_menu(self):
 		self.treeview_local.connect('button_press_event', self._signal_treeview_button_pressed)
 
 		self.popup_menu = Gtk.Menu.new()
 		menu_item = Gtk.CheckMenuItem.new_with_label('Show Hidden Files')
-		menu_item.connect('toggled', self.hidden_files)
-		self.hidden_files = menu_item
+		menu_item.connect('toggled', self.signal_menu_toggled_hidden_files)
+		self.signal_menu_toggled_hidden_files = menu_item
 		self.popup_menu.append(menu_item)
 
 		menu_item = Gtk.MenuItem.new_with_label('Delete')
-		menu_item.connect('activate', self.delete_prompt)
+		menu_item.connect('activate', self.signal_menu_activate_delete_prompt)
 		self.popup_menu.append(menu_item)
 		self.popup_menu.show_all()
 
 	def _signal_treeview_button_pressed(self, _, event):
-		if event.button == 3:
+		if event.button == Gdk.BUTTON_SECONDARY:
 			self.popup_menu.popup(None, None, None, None, event.button, Gtk.get_current_event_time())
 			return True
 		return
 
-	def expand_row(self, _, treeiter, treepath):
-		newPath = self._tvmodel[treeiter][2]  # pylint: disable=unsubscriptable-object
-		self.load_dirs(newPath, treeiter)
+	def signal_expand_row(self, _, treeiter, treepath):
+		new_path = self._tvmodel[treeiter][2]  # pylint: disable=unsubscriptable-object
+		self.load_dirs(new_path, treeiter)
 		self._tvmodel.remove(self._tvmodel.iter_children(treeiter))
 
-	def collapse_row(self, _, treeiter, treepath):
+	def signal_collapse_row(self, _, treeiter, treepath):
 		current = self._tvmodel.iter_children(treeiter)
 		while current:
 			self._tvmodel.remove(current)
 			current = self._tvmodel.iter_children(treeiter)
 		self._tvmodel.append(treeiter, [None, None, None, None, None, None, None])
 
-	def hidden_files(self, _):  # pylint: disable=method-hidden
+	def load_dirs(self, path, parent=None):
+		counter = 0
+		dir_list = self._yield_dir_list(path)
+		for name in dir_list:
```

Comment:
This should be `for name in self._yield_dir_list(path)` once the Local and Remote functions have been updated to act as generators (using the `yield` keyword).